manufacturing-ai-suite/wind-turbine-anomaly-detection/docs/user-guide/Overview.md

```diff
@@ -1,87 +1,8 @@
-# Wind Turbine Anomaly Detection
+# Overview
 
-Time Series predictive maintenance use case to detect the anomalous power generation patterns relative to wind speed for the wind turbines.
+Wind Turbine Anomaly Detection use case to detect the anomalous power generation patterns relative to wind speed for the wind turbines.

```

Comment:
Let's include the time series predictive maintenance somewhere in the sentence as this is the original requirement https://jira.devtools.intel.com/browse/ITEP-23315?filter=418929